src/utils/apiKey.js

```diff
@@ -0,0 +1,2 @@
+const apiKey = '7c4a986b16b3b892bd7111a358d63e05';
+export default apiKey;
```

Comment:
Move this into one file, `utils/mdHelpers`.
```suggestion
const apiKey = '7c4a986b16b3b892bd7111a358d63e05';
const posterUrl = 'https://image.tmdb.org/t/p/w220_and_h330_face/';

export {
  apiKey,
  posterUrl
}
```